viewer_server/client_db_access_handler.py

```diff
@@ -89,11 +89,13 @@ def __init__(self,
                  checker_md_docs,
                  checker_md_docs_map,
                  suppress_handler,
-                 db_version_info):
+                 db_version_info,
+                 version):
```

Comment:
maybe naming `version` to `package_version` or something similar can be more descriptive